templates/web/pages/robots/robots.tt.txt

```diff
@@ -1,9 +1,13 @@
-User-agent: *
+[% IF deny_access %]User-agent: *
+Disallow: /
+[% ELSE %]User-agent: *
```

Comment:
you can use [chomping](https://metacpan.org/dist/Template-Toolkit/view/lib/Template/Manual/Syntax.pod#Chomping-Whitespace) `-%` to make it more readable

```suggestion
[% IF deny_access -%]
User-agent: *
Disallow: /
[% ELSE -%]
User-agent: *
```

…and so one also below.